smart-contracts/contracts/interfaces/IPublicLock.sol

```diff
@@ -304,7 +290,7 @@ interface IPublicLock {
   function getTransferFee(
     address _owner,
     uint _time
-  ) external view returns (uint);
+  ) external view returns (uint); // solium-disable-line indentation
```

Comment:
hmm instead of adding this to each impacted line, could we just do it once for the entire file?  https://ethlint.readthedocs.io/en/latest/user-guide.html#configuring-with-comments

And maybe also include a comment pointing to the bug

Reply:
I could. Although this is repetitive, I didn't want to disable the linting for the whole file if I didn't have to. I don't mind switching it though.
I will add a link to the bug.